nansat/tests/test_exporter.py

```diff
@@ -332,18 +328,15 @@ def test_export_netcdf_complex_remove_meta(self):
         self.assertTrue((n[1] == exported[1]).any())
 
     def test_export_netcdf_arctic(self):
-        ''' Test export of the arctic data without GCPS
-        '''
         n = Nansat(self.test_file_arctic)
         n.export(self.tmpfilename)
         exported = Nansat(self.tmpfilename)
         self.assertTrue((n[1] == exported[1]).any())
         self.assertTrue((n[2] == exported[2]).any())
         self.assertTrue((n[3] == exported[3]).any())
+        time.sleep(1)
```

Comment:
What is a reason for that?

Reply:
unit tests should ideally take miliseconds. Adding a 1 second sleep like this seems to really break that :)

Reply:
That I can also remove. I was experimenting why some files are not released after they are exported in windows. I though if I give some time it would help. Nope :(